src/fsharp/SignatureConformance.fs

```diff
@@ -249,6 +249,11 @@ type Checker(g, amap, denv, remapInfo: SignatureRepackageInfo, checkingSig) =
                   let res = 
                       (implArgInfos,sigArgInfos) ||> List.forall2 (List.forall2 (fun implArgInfo sigArgInfo -> 
                           checkAttribs aenv implArgInfo.Attribs sigArgInfo.Attribs (fun attribs -> 
+                              match implArgInfo.Name, sigArgInfo.Name with 
+                              | Some iname, Some sname when sname.idText <> iname.idText -> 
```

Comment:
should the comparison be locale aware, or be StringCompare.Ordinal?

Reply:
Everywhere else in signature conformance checking we just use default F# string equality i.e. ``System.String.Equals(string1,string2)``, e.g.
```
            if implTycon.LogicalName <> sigTycon.LogicalName then (errorR (Error (FSComp.SR.DefinitionsInSigAndImplNotCompatibleNamesDiffer(implTycon.TypeOrMeasureKind.ToString(),sigTycon.LogicalName,implTycon.LogicalName),m)); false) else
            if implTycon.CompiledName <> sigTycon.CompiledName then (errorR (Error (FSComp.SR.DefinitionsInSigAndImplNotCompatibleNamesDiffer(implTycon.TypeOrMeasureKind.ToString(),sigTycon.CompiledName,implTycon.CompiledName),m)); false) else
```
If we change it in one place we'd need to change it everywhere

Reply:
Yes ... but what is the right thing to do?

It may be that locale aware case sensitive is the right thing to do.  It's not clear to me is all.

Reply:
I think we can't change this at this stage - all the name lookups in the compiler are based on exact string equality.  

Also, part of the intent of using the signature names is to make sure that the .NET/IL/C#/debug and F# views of named arguments are the same.  We don't know what equality check is used by visual studio when you mouse-hover over an identifier and it tries to look it up.  

In any case, it's an orthogonal issue to this PR